CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.9.1 (2021-12-03)
+
+- Fixed the viewkeeper direct to ignore the viewport margin when attaching to a scrollable host. [#279](https://github.com/blackbaud/skyux-core/pull/279)
```

Comment:
Not sure what direct is referring to. Is it a function? If so, should the formatting be different?

Reply:
Should we just delete `direct`?

Reply:
That was a typo. It should have said `directive`. I’ve updated it.